tests/capture_tools_output/test_tools_output.py

```diff
@@ -295,8 +298,6 @@ def test_format_annotations(
                         break
                 else:  # pragma: no cover
                     raise RuntimeError(f"line {line} not in ranges {repr(ranges)}")
-        else:  # pragma: no cover
-            raise RuntimeWarning(f"unrecognized record: {message}")
```

Comment:
Now that the loop is unrolled, not required anymore.

Reply:
![image](https://github.com/cpp-linter/cpp-linter/assets/14963867/babc6c2c-eb8a-4280-bff7-6c6f591fba9d)
This coverage is exactly why that `else: raise` was there. the test is passing because there is nothing to parse. Something is wrong here. Maybe change to `capsys.readouterr().out` (not `.err`)?

Reply:
disregard, coverage report was out of date.